pkg/render/logstorage.go

```diff
@@ -1760,6 +1760,11 @@ func (es *elasticsearchComponent) kibanaAllowTigeraPolicy() *v3.NetworkPolicy {
 			Protocol:    &networkpolicy.TCPProtocol,
 			Destination: networkpolicy.DefaultHelper().ESGatewayEntityRule(),
 		},
+		{
```

Comment:
Does it really need egress access to the Job?

Reply:
Not sure ... I was simply following the pattern used for esgateway.

Reply:
es-gateway shouldn't be making egress calls to the Job AFAIK, so I think we don't need it.